README.md

```diff
@@ -195,11 +195,10 @@ An [Effect] are the instruction to the [Executor] to run set of [Task]s concurre
 
 ```ipldsch
 type Task struct {
-  on      URI
-  call    Ability
-  input   {String : Any}
-
-  nnc     string
+  obj    URI
+  call   Ability
+  args   {String : Any}
+  nnc    string
```

Comment:
as discussed, but should be `Optional`, right @expede.

Reply:
Can we just use `""` if you don't care about `nnc` that way we don't have two logical empty nonce ?

Reply:
Yeah, the Q is if it's actually not there vs there or we also gen a nonce if the incoming one is empty (regarding impl), right?

Reply:
Ah, right, we can just use an empty string. Let's do that for consistency 👍

Reply:
Or what if we made it an `implicit ""`?